code/game/objects/items/devices/radio/headset.dm

```diff
@@ -258,6 +258,9 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	name = "\proper mini Integrated Subspace Transceiver "
 	subspace_transmission = FALSE
 
+/obj/item/radio/headset/silicon/pai/ComponentInitialize()
+	. = ..()
+	AddElement(/datum/element/empprotection, EMP_PROTECT_WIRES)
```

Comment:
pretty sure this means they can _never_ be emped which is an issue because that means they kill any chance of someone trying to stealthily kill someone who has a pAI

the bug is they can't turn their comms back on after being emped, not the fact they can be emped in the first place

Reply:
Funniliy enough, pAIs use another variable for radio shortcuts. see `pai/say.dm`. Also they get silenced upon EMP.